lib/console.js

```diff
@@ -249,6 +249,75 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
+  'symbol', 'undefined', 'boolean', 'number', 'string', 'function',
```

Comment:
`function` works just fine in Chrome: `console.table([function a() {}, function b() {}])`.